.size-limit.js

```diff
@@ -22,7 +22,7 @@ module.exports = [
     name: 'The initial cost paid for using one component with ES modules',
     webpack: true,
     path: 'packages/material-ui/build/esm/Paper/index.js',
-    limit: '17.8 KB',
+    limit: '17.9 KB',
```

Comment:
What happened here? Did you regenerate the whole lockfile again?

Reply:
Yes, I did.

Reply:
Why? `yarn add next` or `yarn upgrade next` would've done the trick. Now we have all sorts of side effects in this PR.

Reply:
Because it's a good thing to stay up to date?

Reply:
> [core] Upgrade to Next.js 8